src/pkg/projects/netstandard/pkg/dir.props

```diff
@@ -3,18 +3,17 @@
   <PropertyGroup>
     <IsFrameworkPackage>true</IsFrameworkPackage>
     <ShortFrameworkName>netstandard</ShortFrameworkName>
-    <ProductBrandPrefix>Microsoft .NET Standard</ProductBrandPrefix>
-
-    <ProductBandVersion>2.1</ProductBandVersion>
-    <ProductionVersion>$(ProductBandVersion).0</ProductionVersion>
   </PropertyGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., dir.props))\dir.props" />
 
   <PropertyGroup>
-    <FrameworkListName>.NET Standard 2.1</FrameworkListName>
+    <ProductBrandPrefix>Microsoft .NET Standard</ProductBrandPrefix>
+    <ProductBandVersion>$(NetStandardProductBandVersion)</ProductBandVersion>
+    <ProductionVersion>$(ProductBandVersion).0</ProductionVersion>
```

Comment:
In the latest mock build, this MSI is generated as 3.0.0 by the new tooling:

```
  Light 'NETStandard.Library.Ref'
  Created 'F:\workspace.1\_work\1\s\bin/win-x64.Release/packages/netstandard-targeting-pack-3.0.0-preview7-27824-03-win-x64.msi'
```

But the nupkg creation is (rightfully) looking for `2.1.0`.

```
File not found: 'F:\workspace.1\_work\1\s\bin/win-x64.Release/packages/netstandard-targeting-pack-2.1.0-preview7-27824-03-win-x64.msi'.
```

I'm pretty sure this is because `ProductionVersion` here was moved below the import where it isn't able to influence the derived props. Trying to use stuff generated by the new project-based infra in the old infra is dicey... to keep this simple, I recommend duplicating the info for now.

The migration to the Arcade SDK is putting version prop generation in targets, so any effort to dedup will have a lot more flexibility then. Further along, the insertion nupkg generation should become part of the new infra or make way for fancier Arcade tooling.

Reply:
Reverting the file to original contents.